src/tick.js

```diff
@@ -125,7 +126,6 @@
       return player._inTimeline;
     });
 
-    pendingEffects.length = 0;
     pendingEffects.push.apply(pendingEffects, newPendingClears);
     pendingEffects.push.apply(pendingEffects, newPendingEffects);
```

Comment:
It seems like we can end up with the same effects repeated multiple times in `pendingEffects`, can we dedupe this list before applying?